src/Microsoft.AspNet.Hosting/Internal/HostingEngine.cs

```diff
@@ -88,9 +88,7 @@ public virtual IApplication Start()
             var logger = _applicationServices.GetRequiredService<ILogger<HostingEngine>>();
             var contextFactory = _applicationServices.GetRequiredService<IHttpContextFactory>();
             var contextAccessor = _applicationServices.GetRequiredService<IHttpContextAccessor>();
-#pragma warning disable 0618
```

Comment:
Do a quick search for `0618` and make sure you caught all of these (if you didn't already)